google-cloud-bigquery/src/main/java/com/google/cloud/bigquery/spi/v2/HttpBigQueryRpc.java

```diff
@@ -230,12 +303,36 @@ public Table createSkipExceptionTranslation(Table table, Map<Option, ?> options)
     // unset the type, as it is output only
     table.setType(null);
     TableReference reference = table.getTableReference();
-    return bigquery
-        .tables()
-        .insert(reference.getProjectId(), reference.getDatasetId(), table)
-        .setPrettyPrint(false)
-        .setFields(Option.FIELDS.getString(options))
-        .execute();
+    Bigquery.Tables.Insert bqCreateRequest =
+        bigquery
+            .tables()
+            .insert(reference.getProjectId(), reference.getDatasetId(), table)
+            .setPrettyPrint(false)
+            .setFields(Option.FIELDS.getString(options));
+
+    Span createTable = null;
+    if (this.options.isOpenTelemetryTracingEnabled()
+        && this.options.getOpenTelemetryTracer() != null) {
+      createTable =
+          this.options
+              .getOpenTelemetryTracer()
+              .spanBuilder("com.google.cloud.bigquery.BigQueryRpc.createTable")
+              .setSpanKind(SpanKind.CLIENT)
+              .setAttribute("bq.rpc.service", "TableService")
+              .setAttribute("bq.rpc.method", "InsertTable")
+              .setAttribute("bq.rpc.system", "http")
+              .setAttribute("bq.rpc.access_token", bqCreateRequest.getAccessToken())
+              .setAttribute("bq.rpc.oauth_token", bqCreateRequest.getOauthToken())
+              .setAttribute("bq.rpc.request_method", bqCreateRequest.getRequestMethod())
+              .setAllAttributes(otelAttributesFromOptions(options))
+              .startSpan();
+    }
+    Table tableResponse = bqCreateRequest.execute();
+    if (createTable != null) {
+      createTable.setAttribute("bq.rpc.response", tableResponse.toPrettyString());
```

Comment:
My main worry about logging the responses is going to be large responses.  In general, should we enforce a configurable limit?